_posts/2023-06-16-childcare-lactation.md

```diff
@@ -0,0 +1,24 @@
+---
+author: Jason Judkins, Metta Rolando
+category: General
+date: 2023-06-16 12:00:00
+image: /static/img/blog/childcare-lactation-family.png
+layout: post
+post_photo_alt: DjangoCon US 2023 Childcare, Lactation, and Family
+post_photo_url: /static/img/blog/childcare-lactation-family.png
+title: WANTED! Parents at DjangoCon US 2023!
+---
+
+# Are you a working parent? Are you a nursing parent? We want you to attend DjangoCon US! 
+
+## Inclusion in the DjangoCon US space matters! 
+
+## We want you to know we can help!
+
+We can assist in connecting you with a local daycare facility, and we will reimburse any child care expenses up to $150 per day.
+
+For nursing parents, we will provide a clean, comfortable, and private lactation space including a small refrigerator to store pumped milk, labeling supplies, and antibacterial wipes.
+
+## Want more information?
+
+Please contact us at [hello@defna.org](mailto:hello@defna.org) if you are interested in utilizing DjangoCon US’s child care reimbursement or lactation space services, so we can be sure we meet your needs and help you attend DjangoCon US 2023 in Durham this October!
```

Comment:
This sentence is a bit long. Consider making it two separate statements. Something like "Let us know how we can meet your needs and help you attend DjangoCon US 2023 in Durham this October! Please contact us at [hello@defna.org](mailto:hello@defna.org) if you are interested in utilizing DjangoCon US’s child care reimbursement or lactation space services."

Or reverse them. "Contact us" first. Then "let us know how we can meet your needs."